### ISSUE TITLE
predicted frequencies behave oddly after filtering

### ISSUE THREAD

**Issue by rneher** [nextstrain/auspice#838]
When you filter by region on nextstrain.org/flu, the frequency panel looks like this

![image](https://user-images.githubusercontent.com/8379168/71288934-2d335380-236c-11ea-9bd8-ac73b2952f94.png)

This is some interaction between the normalization of frequencies (see issue #784) and the way the predicted frequencies are stored and extrapolated. @huddlej -- do you have a suggestion on how to improve this behavior?

**Comment by huddlej** [comment]
I'm not immediately sure why the predicted frequencies would be treated any differently, since observed and projected frequencies are stored in the same arrays. I am not sure where to start looking in auspice for the code that does this normalization: searches for "normalise", "normalize", "normalization", and "normalisation" all return discussions about normalizing to 100%.

Maybe [this code related to changing y max is relevant](https://github.com/nextstrain/auspice/blob/9b85067/src/components/frequencies/index.js#L63-L66)?

If @jameshadfield or @trvrb could point me to where the visualization code lives, I could look into the interaction with the tip frequencies data themselves.

**Comment by jameshadfield** [comment]
I'm not sure this is an auspice bug, but it may well be & I'm happy to dig into the code if so. 

The non-predicted frequencies are a-priori set to be constant over time (hence the stream graph max y-value is constant before the dashed line) but there's no reason the predicted frequency of a region should be constant in the future (more of a question for @huddlej). E.g. I interpret this screenshot to say that "european" strains (i.e. strains observed in europe & used as input for the model) are going to increase in worldwide prevalence. 

![image](https://user-images.githubusercontent.com/8350992/71328866-bb812280-2582-11ea-9c93-041f57d30fc3.png)

**Comment by trvrb** [comment]
This is not an Auspice bug. It's the correct display of how the fitness model projects frequencies in time. But this reveals deficiencies in having just a single global strain frequencies estimate.

I had noticed this before, and looked into just renormalizing everything in Auspice, but this had its own issues. I'll try to expand on this shortly.

**Comment by rneher** [comment]
we might want to reconsider the overall normalization or provide a toggle and make it the default.

**Comment by rneher** [comment]
I agree -- it is not a bug. More a matter of what we think the predictions should convey (and I am admittedly quite fuzzy on that).

**Comment by huddlej** [comment]
Thank you, @jameshadfield and @trvrb for clarifying! I misinterpreted the original question as a problem with filtering in general. It totally makes sense that this is only happening for filter by region.

I suspect that users would be more surprised to find that each region has a fixed maximum frequency when none of the other available filters behave the same way. But this normalization is determined by both the [strain sampling logic](https://github.com/nextstrain/seasonal-flu/blob/a86b4b0/scripts/select_strains.py#L114-L116) and the [regional weighting in KDE frequencies](https://github.com/nextstrain/seasonal-flu/blob/82bd93d/Snakefile_base#L600). Would this make a front-end toggle trickier?

Maybe we could discuss this at our next Nextstrain meeting or on Slack?

**Comment by huddlej** [comment]
This was resolved by #1158
